packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec

```diff
@@ -1,17 +1,22 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
+#
```

Comment:
```suggestion
```

Reply:
Blast, I squished the wrong commit into the wrong commit.